README.md

```diff
@@ -136,22 +143,22 @@ contract HashPairTest {
 }
 ```
 
-This solidity file is missing the tests for `When first arg is bigger than second arg`,
+This Solidity file is missing the tests for `When first arg is bigger than second arg`,
 which would be reported after running `bulloak check`, like so:
 
 ```text
-Codegen not found: Couldn't find a corresponding element for "whenFirstArgIsBiggerThanSecondArg" in the solidity file.
-Codegen not found: Couldn't find a corresponding element for "test_WhenFirstArgIsBiggerThanSecondArg" in the solidity file.
+Codegen not found: Couldn't find a corresponding element for "whenFirstArgIsBiggerThanSecondArg" in the Solidity file.
+Codegen not found: Couldn't find a corresponding element for "test_WhenFirstArgIsBiggerThanSecondArg" in the Solidity file.
```

Comment:
I'll change it in the following commit so no worries, but note this is generated by bulloak!